core/state/statedb.go

```diff
@@ -298,7 +298,6 @@ func (s *StateDB) SubRefund(gas uint64) {
 }
 
 // Exist reports whether the given account address exists in the state.
-// Notably this also returns true for self-destructed accounts.
```

Comment:
You can extend the comment by mentioning `self-destructed accounts within the current transaction.`

Reply:
Thanks, updated